### PR TITLE
R4 Document Reference  create and update workflow updation

### PR DESCRIPTION
**Pull request by akashdeep677**
Description
----
R4 Document Reference  create and update workflow updation

PR Checklist
----
- [x] Screenshot(s) of changes attached before changes merged.
- [ ] Screenshot(s) of changes attached after changes merged and published.

### CODE REVIEW

**Comment by akashdeep677** [comment]
Screenshot 2022-11-04 at 5 58 12 PM
Screenshot 2022-11-04 at 5 55 55 PM
Screenshot 2022-11-04 at 5 56 04 PM
Screenshot 2022-11-04 at 5 56 15 PM
Screenshot 2022-11-04 at 5 56 28 PM
Screenshot 2022-11-04 at 5 56 46 PM
Screenshot 2022-11-04 at 5 56 56 PM
Screenshot 2022-11-04 at 5 57 10 PM
Screenshot 2022-11-04 at 5 57 27 PM
Screenshot 2022-11-04 at 5 57 35 PM
Screenshot 2022-11-04 at 5 57 41 PM
Screenshot 2022-11-04 at 5 57 56 PM
Screenshot 2022-11-04 at 5 58 03 PM

**Review by aki-mathur — APPROVED** [review]

**Review by Surya-Rnsit — APPROVED** [review]

File: lib/resources/r4/document_reference.yaml [hunk 1]
```diff
@@ -166,7 +166,7 @@ fields:
           "attachment": {
             "contentType": "text/xml;charset=utf-8",
             "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
-            "title": "Summary Note",
+            "title": "Height Weight Allergy Rule",
```
Comment (SA030800) [inline lib/resources/r4/document_reference.yaml:169]: isn't more appropriate to use 'Admission Note Physician' here instead?

File: lib/resources/r4/document_reference.yaml [hunk 2]
```diff
@@ -186,8 +186,8 @@ fields:
      "attachment": {
             "contentType": "application/xml;charset=utf-8",
             "data": "PGh0bWw+Cjx0aXRsZT4gVGVzdCBEb2N1bWVudCA8L3RpdGxlPgoKRG9jdW1lbnQgY29udGVudCEKCjwvaHRtbD4=",
-            "title": "Summary Note",
-            "creation": "2020-01-15T14:35:29.000Z"
+            "title": "Height Weight Allergy Rule",
```
Comment (SA030800) [inline lib/resources/r4/document_reference.yaml:189]: same here???

Comment (akashdeep677) [reply lib/resources/r4/document_reference.yaml:189]: I will check with my team lead and update you. Thanks for input.

**Comment by Surya-Rnsit** [comment]
changes merged and published.
Screenshot 2022-12-01 at 6 37 34 PM
Screenshot 2022-12-01 at 6 38 12 PM

**Pull request merged**